wls-basisdaten-service/pom.xml

```diff
@@ -393,7 +393,7 @@
                             <goal>generate</goal>
                         </goals>
                         <configuration>
-                            <inputSpec>${project.basedir}/src/main/resources/openapis/openapi.json</inputSpec>
+                            <inputSpec>${project.basedir}/src/main/resources/openapis/openapi.eai.0.1.1-Beta1.json</inputSpec>
```

Comment:
bitte keine Beta verwenden auf einen Branch der nach dev gehen soll.

Reply:
ich musste doch den Inhalt aus dem Release nehmen und hier einspielen. Dann sollte ich den Datei-Namen auch mit der Version versehen, für Übersichtlichkeit. Der Name der Datei habe ich aus der Datei selbst genommen. Und die Datei wie gesagt aus [dem letzten Release des eai-Services](https://github.com/it-at-m/Wahllokalsystem/releases/download/wls-eai-service%2F0.1.1-Beta1/openapi.json) 
Hätte ich von wo anders das Asset nehmen oder hätte ich einfach den Suffix "Beta" nicht schreiben sollen?

Reply:
Bitte die API aus https://github.com/it-at-m/Wahllokalsystem/releases/tag/wls-eai-service%2F0.1.0 verwenden. Das andere Release war im Rahmen einer Workflowanpassen zu Testzwecken erzeugt wurden. Es kann sein dass sie gleich ist, aber muss nicht sein.